### ISSUE TITLE
Dutch names bumblebees

### ISSUE THREAD

**Issue by chrisvanswaay** [BiologicalRecordsCentre/ABLE#709]
Would it be possible to add these Dutch names of bumblebees to the app? I think I asked for this earlier, but can't find the topic anymore, and get complaints from recorders who want to enter bumblebees.
@kazlauskis @CrisSevilleja 
[Uploading 202011_Soortenlijst Hommels NL_.xlsx…]()

**Comment by chrisvanswaay** [comment]
Just noticed that the Dutch bumblebee names do appear on the website:
![image](https://github.com/user-attachments/assets/9f510bb9-d5d8-4e31-9427-36ae2e295073)

So they are in the system somewhere. However they do NOT show up on the app, so all bumblebees have to be entered by their scientific name. 
@kazlauskis Hope this can be adapted.

**Comment by DavidRoy** [comment]
@JimBacon can you add in these names so they can be included in the next app update

**Comment by JimBacon** [comment]
Looks to me like they were added to the database two and a half years ago, https://github.com/BiologicalRecordsCentre/ABLE/issues/411. 

I expect there is something going wrong with how we extract the bumblebee names for the app.
@kazlauskis can you tell me what you do to get bee names so I can investigate?

**Comment by chrisvanswaay** [comment]
Indeed, the names are already in the database, but you cannot use them when entering bumblebees in the app, only scientific names. Would be great if this could be solved.
